cloud_Azure/terraform/module/tests/main.tf

```diff
@@ -1,17 +1,32 @@
 terraform {
   required_version = ">= 0.12.0"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.85.0"
+    }
+    kentik-cloudexport = {
+      source  = "kentik/kentik-cloudexport"
+      version = ">= 0.2.0"
```

Comment:
Let's require the latest and greatest (>= 0.4.1), so that we eventually don't have to deal with historic problems.